index.html

```diff
@@ -19,9 +20,21 @@
     <div class="menu-icon-container">
       <img src="toolbar-icon.png" alt="Menu icon">
     </div>
-
   </header>
 
+  <section class="headline">
+    <h1>I’m Mariane Glad to see you!</h1>
+    <p class="intro">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <p class="connect">LET’S CONNECT</p>
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you make use of a header tag from `h2` to `h6` for your `LET'S CONNECT` text because it's a header.